### DISCUSSION TITLE
Install my fork of the awesome inertia package

### DISCUSSION THREAD

**Discussion by StefanNeuser** [inertiajs/inertia#537] [Help] [answered]
Hello everybody,

it's driving me nuts. I forked the repo, made changes, start a Pull-Request and now i don't get it how to install my form via yarn or npm. I dont hava a idea how to tell my package.json (npm) the use inertia and inertia-react from the sub-directory structure in this repo like packages/...

Please help me or even better, approve my PR :-)

**Comment by claudiodekker** [comment, marked as answer]
Hi,

This isn't really an Inertia issue/question, but is rather a question related to using NPM with your own package(s), so I'm afraid this isn't really the right place for it. Regardless, here's a thread with some ideas on how you could (hopefully) solve this: https://github.com/cyclejs/cyclejs/issues/550#issuecomment-285891571

If that doesn't help, I recommend searching the internet for something like "npm install forked package from github monorepo", or asking your question over on the [Github Community forums](https://github.community/c/software-development/47) 

Good luck!
